articles/if-you-like-it-then-you-shouldnt-put-some-code-in-it.md

```diff
@@ -0,0 +1,37 @@
+---
+authorName: Jonathan Rothwell
+authorGithubUsername: jrothwell
+issue: 94
+title: If you like it then you shouldn't put some code in it
+---
+# {{page.title}}
+
+In 2017, a crowdfunding campaign from Australia claimed that “in a world of technical overload there is sometimes no space to fit the simple things on your \[bicycle\] handlebars.” They wanted AU$85,000 to manufacture ‘the simple bike bell’.
+
+The ‘simple’ bike bell worked like this: Instead of of striking an actual clapper against a metal bell, you have a small button which you mount on your handlebars. You pair the button with your phone (which must run a supported operating system.) When you want to use the bell, you press the Bluetooth button. This sends a message to your phone (hopefully charged, within range, with Bluetooth turned on, and with the manufacturer’s app open and running.) Provided these conditions are all met, your phone’s speaker plays the sound of a bicycle bell... probably. What could be simpler than that? (Kickstarter users only pledged 1.7% of its funding target.)
+
+What are the thought processes that lead to trashy or tasteless IoT solutions like this? It’s simple: someone somewhere sees an itch, which they scratch with what they know—technology. They then pat themselves on the back for being so ‘disruptive’, make a few news releases, and voilà: profit. (Maybe.)
```

Comment:
Should it be all 1 word?  „what-they-know-technology“
In German the whole dash usage is different (broken in my eyes, since I dont understand it :)

Reply:
This should be a colon. Will fix.